### ISSUE TITLE
textdistance version doesn't match

### ISSUE THREAD

**Issue by liyunlong0516** [spyder-ide/spyder#20738]
### What steps will reproduce the problem?

I was installed Spyder V5.1.5 single，but i want to add another module tensorflow, so i try to config tools -> PYTHONPATH, adding ${PYTHON_HOME}/Lib/site-packages directory and then alter this error.

### Traceback
```python-traceback
Traceback (most recent call last):
  File "D:\ProgramFiles\Spyder515\pkgs\spyder\plugins\editor\widgets\codeeditor.py", line 1915, in finish_code_folding
    folding_panel.update_folding(self._folding_info)
AttributeError: 'CodeEditor' object has no attribute '_folding_info'
Traceback (most recent call last):
  File "D:\ProgramFiles\Spyder515\pkgs\spyder\plugins\editor\widgets\codeeditor.py", line 1915, in finish_code_folding
    folding_panel.update_folding(self._folding_info)
AttributeError: 'CodeEditor' object has no attribute '_folding_info'
```

## Versions

* Spyder version: 5.1.5 
* Python version: 3.7.9
* Qt version: 5.12.10
* PyQt5 version: 5.12.3
* Operating System: Windows 7

### Dependencies

```

# Mandatory:
atomicwrites >=1.2.0          :  1.4.0 (OK)
chardet >=2.0.0               :  4.0.0 (OK)
cloudpickle >=0.5.0           :  2.0.0 (OK)
cookiecutter >=1.6.0          :  1.7.3 (OK)
diff_match_patch >=20181111   :  20200713 (OK)
intervaltree                  :  None (OK)
IPython >=7.6.0               :  7.27.0 (OK)
jedi >=0.17.2;<0.19.0         :  0.18.0 (OK)
jsonschema >=3.2.0            :  3.2.0 (OK)
keyring >=17.0.0              :  23.2.1 (OK)
nbconvert >=4.0               :  6.1.0 (OK)
numpydoc >=0.6.0              :  1.1.0 (OK)
paramiko >=2.4.0              :  2.7.2 (OK)
parso >=0.7.0;<0.9.0          :  0.8.2 (OK)
pexpect >=4.4.0               :  4.8.0 (OK)
pickleshare >=0.4             :  0.7.5 (OK)
psutil >=5.3                  :  5.8.0 (OK)
pygments >=2.0                :  2.10.0 (OK)
pylint >=2.5.0;<2.10.0        :  2.9.6 (OK)
pyls_spyder >=0.4.0           :  0.4.0 (OK)
pylsp >=1.2.2;<1.3.0          :  1.2.2 (OK)
pylsp_black >=1.0.0           :  None (OK)
qdarkstyle =3.0.2             :  3.0.2 (OK)
qstylizer >=0.1.10            :  0.2.1 (OK)
qtawesome >=1.0.2             :  1.0.3 (OK)
qtconsole >=5.1.0             :  5.1.1 (OK)
qtpy >=1.5.0                  :  1.11.1 (OK)
rtree >=0.9.7                 :  0.9.7 (OK)
setuptools >=49.6.0           :  58.0.4 (OK)
sphinx >=0.6.6                :  4.2.0 (OK)
spyder_kernels >=2.1.1;<2.2.0 :  2.1.1 (OK)
textdistance >=4.2.0          :  None (NOK)
three_merge >=0.1.1           :  0.1.1 (OK)
watchdog                      :  2.1.5 (OK)
zmq >=17                      :  22.2.1 (OK)

# Optional:
cython >=0.21                 :  0.29.24 (OK)
matplotlib >=2.0.0            :  None (NOK)
numpy >=1.7                   :  None (NOK)
pandas >=1.1.1                :  None (NOK)
scipy >=0.17.0                :  None (NOK)
sympy >=0.7.3                 :  1.8 (OK)
```

**Comment by dalthviz** [comment]
Hi @liyunlong0516 thank you for the feedback! Seems like this could be a duplicate of https://github.com/spyder-ide/spyder/issues/19514 . A fix for that issue is available starting with Spyder 5.4.0, the latest Spyder release is 5.4.2. Could it be possible for you to update to the latest release?

Also, seems like you want to use `tensorflow` from a different environment than the one where you have Spyder installed and you are launching it, right? The supported way to do that is explained at the Spyder docs FAQ page: https://docs.spyder-ide.org/current/faq.html#using-spyder

Let us know if the info above and updating helps!

**Comment by dalthviz** [comment]
Closing as a duplicate of https://github.com/spyder-ide/spyder/issues/19514
